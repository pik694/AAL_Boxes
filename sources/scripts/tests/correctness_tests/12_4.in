12
9 32 83
38 27 100
25 14 96
93 32 56
91 32 16
86 90 13
16 31 79
91 21 74
81 2 76
16 36 22
30 70 96
82 86 93

