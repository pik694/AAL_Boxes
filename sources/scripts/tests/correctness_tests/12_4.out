8 9 7 11 0 5 2 1 6 3 4 10 
1536990->1239624
